packages/ui5-middleware-fe-mockserver/README.md

```diff
@@ -40,24 +40,24 @@ In order to use the mock server, the npm module `@sap/ux-ui5-fe-mockserver-middl
 
 specVersion: '2.0'
 metadata:
-    name: <NAME>
+  name: <NAME>
 type: application
 server:
-    customMiddleware:
-        - name: sap-fe-mockserver
-          mountPath: /
-          afterMiddleware: compression
-          configuration:
-            annotations:
-              localPath: './webapp/annotations/annotation.xml'
-              urlPath: '/sap/opu/odata/IWFND/CATALOGSERVICE;v=2/Annotations*'
-            services:
-              - urlPath: '/sap/opu/odata/sap/<SERVICE_NAME>'
-              metadataPath: './webapp/localService/metadata.xml'
-              mockdataPath: './webapp/localService/data'
-              - urlPath: '/sap/opu/odata/sap/<OTHER_SERVICE_NAME>'
-              metadataPath: './webapp/localService/other_metadata.xml'
-              mockdataPath: './webapp/localService/data'
+  customMiddleware:
+    - name: sap-fe-mockserver
+      mountPath: /
+      afterMiddleware: compression
+      configuration:
+        annotations:
+          - localPath: './webapp/localService/myServiceAnnotation.xml'
+          urlPath: '/sap/opu/odata/IWFND/CATALOGSERVICE;v=2/Annotations*'
```

Comment:
Shouldn't that be one tab more? (I hate yaml btw)

Reply:
possibly yes, i think both work... let me adjust i also hate yaml :)